Formula/c/colmap.rb

```diff
@@ -36,10 +36,6 @@ class Colmap < Formula
   def install
     ENV.append_path "CMAKE_PREFIX_PATH", Formula["qt@5"].prefix
 
-    # Use C++17 for compatibility with ceres-solver >= 2.2.0.
-    # Issue ref: https://github.com/colmap/colmap/issues/2247
-    inreplace "src/CMakeLists.txt", "-std=c++14", "-std=c++17"
-
```

Comment:
Hoping that this is no longer required due to https://github.com/colmap/colmap/pull/2012/files

Reply:
yeah, the PR got included in the 3.9 release.